CHANGES.rst

```diff
@@ -6,7 +6,7 @@ For changes before version 3.0, see ``HISTORY.rst``.
 5.5 (unreleased)
 ----------------
 
-- Nothing changed yet.
+- Switch from ``-Ofast`` to ``-O3`` when compiling code for Linux wheels.
```

Comment:
Maybe include a link somewhere that explains why this is important?  E.g. to this pull request.

Reply:
Fixed in https://github.com/zopefoundation/AccessControl/pull/133/commits/4708b9f2787de5b53b772ff5221d3f0326c9b68f